module/VuFindSearch/tests/unit-tests/src/VuFindTest/SearchServiceTest.php

```diff
@@ -781,7 +781,7 @@ public function __construct($backendMock)
      *
      * @return Service
      */
-    protected function resolve($backend, $args)
+    protected function resolve($backendId, $args)
```

Comment:
I'm not sure why this SearchServiceMock class even exists -- it would be better addressed with "normal" PHPUnit mocks. I've revised this test to eliminate it, but unfortunately I don't have permission to push my changes back to your repo. Right now, I'm running the full test suite; if everything passes, I'll just go ahead and merge everything to dev manually and then close this PR.